mosaic-runtime/src/commonTest/kotlin/com/jakewharton/mosaic/layout/PaddingTest.kt

```diff
@@ -0,0 +1,313 @@
+package com.jakewharton.mosaic.layout
+
+import androidx.compose.runtime.Composable
+import assertk.assertThat
+import assertk.assertions.isEqualTo
+import com.jakewharton.mosaic.Container
+import com.jakewharton.mosaic.TestChar
+import com.jakewharton.mosaic.TestFiller
+import com.jakewharton.mosaic.modifier.Modifier
+import com.jakewharton.mosaic.renderMosaic
+import com.jakewharton.mosaic.s
+import com.jakewharton.mosaic.testIntrinsics
+import com.jakewharton.mosaic.ui.Layout
+import com.jakewharton.mosaic.ui.unit.Constraints
+import kotlin.test.Test
+import kotlin.test.assertFails
+
+class PaddingTest {
+	@Test fun paddingAllEqualsToPaddingWithExplicitSides() {
+		assertThat(Modifier.padding(10))
+			.isEqualTo(Modifier.padding(10, 10, 10, 10))
+	}
+
+	@Test fun paddingSymmerticEqualsToPaddingWithExplicitSides() {
+		assertThat(Modifier.padding(10, 20))
+			.isEqualTo(Modifier.padding(10, 20, 10, 20))
+	}
+
+	@Test fun paddingLeftNegative() {
+		assertFails {
+			renderMosaic {
+				SingleFiller(Modifier.padding(left = -2))
+			}
+		}
```

Comment:
For this and the other invalidate argument tests, the inclusion of the render function and a composable seems redundant.
```suggestion
		assertFails {
			Modifier.padding(left = -2)
		}
```